Cargo.toml

```diff
@@ -4,6 +4,7 @@ members = [
     "libsignal-service-actix",
     "libsignal-service-hyper",
 ]
+default-members = ["libsignal-service", "libsignal-service-hyper"]
```

Comment:
I'm sorry, I had to make a choice (I thread a 🪙)

Reply:
Hmm, if you use `unsend-futures`, can't you have all three?

Reply:
It's not really used actually, this is just a 💩 trick to have `cargo check` do... something.